tests/dummy/app/pods/radio/controller.js

```diff
@@ -15,7 +15,7 @@ export default Controller.extend({
     {label: 'e', value: 'e'}
   ],
 
-  model: Ember.Object.create({
+  model: EmberObject.create({
```

Comment:
I would prefer that we not do this since we only have one Object library right now.